src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AppointmentDtos.kt

```diff
@@ -10,9 +10,6 @@ import java.time.LocalDateTime
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class CourtLocationsResponse(val courtLocations: List<String>? = emptyList())
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
-data class CourtIdsResponse(val courtIds: Set<String>? = emptySet())
```

Comment:
Did you mean to remove this?

Reply:
Yeah, IntelliJ was giving a warning that it was unused